cli/azd/pkg/ux/init_step.go

```diff
@@ -0,0 +1,37 @@
+package ux
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/fatih/color"
+)
+
+type initStep struct {
+	title       string
+	description string
+}
+
+func (s *initStep) Execute(ctx context.Context, stepCtx StepContext) error {
+	c := color.New(color.FgWhite).Add(color.Bold)
+	c.Println(s.title)
+
+	if s.description != "" {
+		fmt.Println(s.description)
+	}
+
+	fmt.Println()
+
+	return nil
+}
+
+func (s *initStep) ContinueOnError() bool {
+	return true
+}
+
+func NewInitStep(title string, description string) Step {
```

Comment:
Consider not exporting this?  It feels like we don't intend code outside of this package to create this step directly.

Reply:
Agreed - don't want to expose the inner complexity of the trival steps that aren't intended to be reused.  Will see what we can do here.